includes/admin/class-sensei-learners-admin-bulk-actions-view.php

```diff
@@ -478,7 +480,7 @@ private function get_learner_courses_html( $user_id ) {
 		$visible_courses = implode( '', array_slice( $html_items, 0, $visible_count ) );
 		$hidden_courses  = implode( '', array_slice( $html_items, $visible_count ) );
 
-		return $visible_courses . '<div class="sensei-students__enrolled-courses-detail hidden">' . $hidden_courses . '</div>' . $more_button;
+		return $visible_courses . '<div class="sensei-students__enrolled-courses-detail">' . $hidden_courses . '</div>' . $more_button;
```

Comment:
Because visible courses and the courses per page are now both `3`, the `$hidden_courses` will always be empty. So we can get rid of it.